portfolio/src/main/webapp/index.html

```diff
@@ -90,10 +90,31 @@ <h5 class="text-white h4">Menu</h5><br>
     <br><br><br>
   </div>
 
+  <div class="row peach">
+    <div class="col-2">
+    </div>
+    <div class="col-8">
+      <hr>
+    </div>
+  </div>
+
   <div class="row peach">
 	<div class="col center-text">
-      <button class="btn btn-outline-primary" onclick="getHello()">Fetch the hello!</button>
-      <div id="hello-container"></div>
+      <button class="btn btn-outline-primary" onclick="getComments()">Fetch the comments!</button>
+      <div id="comments-container"></div>
+	</div>
+  </div>
+
+  <div class="row peach">
+	<div class="col-3">
+	</div>
+	<div class="col-6 center-text">
+	  <form action="/data" method="POST">
+	  <ul id="history"></ul>
+      Enter a comment!<br/>
```

Comment:
I think you need a `</div>` here.  Most browsers will display HTML even if the tags aren't properly closed, but you might run into some unexpected issues (for example, the `#text-input` here is inside of the `.col-6` div - which may not be what you want

Reply:
I think there is </div> a couple lines down, right after the submit button. If I close the tag earlier, then the text box spans the whole width of the page. Would you like me to change it to this, or is there a way to maintain the format while moving the closing </div> up?

Reply:
OK, this might just be an indentation issue then.  Once the idents are consistent (changed to spaces) it will probably be easier to tell where the closing tags are